src/lib/Scenes/Artwork/Components/AboutArtist.tsx

```diff
@@ -26,9 +26,11 @@ export class AboutArtist extends React.Component<AboutArtistProps> {
           <Sans size="3t" weight="medium" mb={2}>
             {hasSingleArtist ? "About the artist" : "About the artists"}
           </Sans>
-          {artists.map(artist => (
-            <ArtistListItem key={artist.id} artist={artist} />
-          ))}
+          <Join separator={<Spacer my={1} />}>
```

Comment:
One last try-- I think this makes the most sense! When there are multiple artists, we need some spacing between them.

Reply:
looks good!!